product/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+class Category(models.Model):
+    name                = models.CharField(max_length = 45)
+    description         = models.TextField
+    ingredients_image   = models.CharField(max_length = 1000)
```

Comment:
```suggestion
    ingredients_image_url   = models.URLField(max_length = 2000)
```
imageURL 을 담는 필드이므로 정확히 명시해주세요. URL주소를 저장할때에는 URLField를 사용해주세요. max_length도 2000 정도로 넉넉히 주세요.

Reply:
넵!